packages/java-edition/src/mcfunction/parser/argument.ts

```diff
@@ -690,6 +689,19 @@ function range(
 	)
 }
 
+function selectorPrefix(): core.InfallibleParser<core.LiteralNode> {
+	return (src: core.Source): core.LiteralNode => {
```

Comment:
This parse function should check the allowed selector variables, for reference look at `mcfunction/src/parser/literal.ts`